R/bayesiannetworkanalysis.R

```diff
@@ -1174,3 +1269,208 @@ centrality <- function(network, measures = c("closeness", "betweenness", "streng
 
   return(centralityOutput)
 }
+
+
+# A copy of .networkAnalysisNetworkPlot since we changed "estimator" to "model"
+# for now it is the same function, but I will tidy it up later, since it makes no sense to have conditions like
+# "model === mgm"
```

Comment:
Why did you needed to copy this function? Because you do `options[["model"]]` instead of `options[["estimator"]]`? If so, it would be fine to adjust the original function to give it a new argument which in the frequentist case is set to `options[["estimator"]]` and in the Bayesian case to `options[["model"]]`.